### DISCUSSION TITLE
请问怎么设置多路复用

### DISCUSSION THREAD

**Discussion by Xx0101** [eycorsican/leaf#81] [Q&A]
您好,因为我trojan的配置需要配置tls,不过放在哪里不太清楚

**Comment by eycorsican** [comment]
amux 配置参考这里：https://github.com/eycorsican/leaf/blob/master/leaf/tests/test_ws_amux_trojan.rs

leaf 不支持 tls inbound (但支持 tls outbound)，这一块你只能用 nginx 之类的工具来处理。
